config/v1alpha1/tests/insightsdatagathers.config.openshift.io/InsightsConfig.yaml

```diff
@@ -14,3 +14,145 @@ tests:
+        kind: InsightsDataGather
+        spec:
+          gatherConfig:
+            dataPolicy: None
+            storageSpec:
+              type: Ephemeral
+    - name: Should be able to create InsightsDataGather with valid DisabledGatherers names
+      initial: |
+        apiVersion: config.openshift.io/v1alpha1
+        kind: InsightsDataGather
+        spec:
+          gatherConfig:
+            disabledGatherers:
+              - gatherer
+              - valid_gatherer
+              - gatherer/function
+              - gatherer_a/function_a
+      expected: |
+        apiVersion: config.openshift.io/v1alpha1
+        kind: InsightsDataGather
+        spec:
+          gatherConfig:
+            disabledGatherers:
+              - gatherer
+              - valid_gatherer
+              - gatherer/function
+              - gatherer_a/function_a
+    - name: When storageSpec.type is PersistentVolumeClaim then PersistentVolume must be present
+      initial: |
+        apiVersion: config.openshift.io/v1alpha1
+        kind: InsightsDataGather
+        spec:
+          gatherConfig:
+            storageSpec:
+              type: PersistentVolumeClaim
+      expectedError: 'spec.gatherConfig.storageSpec: Invalid value: "object": persistentVolume is required when type is PersistentVolumeClaim, and forbidden otherwise'
+    - name: When storageSpec.type is not PersistentVolumeClaim then PersistentVolume must not be present
+      initial: |
+        apiVersion: config.openshift.io/v1alpha1
+        kind: InsightsDataGather
+        spec:
+          gatherConfig:
+            storageSpec:
+              type: Ephemeral
+              persistentVolume:
+                persistentVolumeClaim:
+                  name: test-claim
+      expectedError: 'spec.gatherConfig.storageSpec: Invalid value: "object": persistentVolume is required when type is PersistentVolumeClaim, and forbidden otherwise'
+    - name: mountPath must not contain colon
+      initial: |
+        apiVersion: config.openshift.io/v1alpha1
+        kind: InsightsDataGather
+        spec:
+          gatherConfig:
+            storageSpec:
+              type: PersistentVolumeClaim
+              persistentVolume:
+                mountPath: /data:/data
+                persistentVolumeClaim:
+                  name: test-claim
+      expectedError: 'spec.gatherConfig.storageSpec.persistentVolume.mountPath: Invalid value: "string": mountPath must not contain a colon'
+    - name: invalid disabledGatherers name
+      initial: |
+        apiVersion: config.openshift.io/v1alpha1
+        kind: InsightsDataGather
+        spec:
+          gatherConfig:
+            disabledGatherers:
+              - invalid_gatherer_1
+      expectedError: 'Invalid value: "string": disabledGatherer must be in the format of {gatherer}/{function} where the gatherer and function are lowercase strings that may include underscores (_) and are separated by a forward slash (/) if the function is provided'
```

Comment:
Reading this value, and the error description, I cannot tell why this is invalid

Reply:
The issue here is the number in the name, I will change the `lowercase strings` to `lowercase letters only` that should be more clear imo